app/controllers/pages_controller.rb

```diff
@@ -31,6 +31,19 @@ def events
 
   def sponsors
     @sponsors = Sponsor.all.where(featured: true)
+    @downloader = Downloader.new
+  end
+
+  def createDownloader
```

Comment:
Should consider using ajax for this so that the user can still stay on the same sponsor page but we would still be able to send file/send feedback to user.